src/main/java/com/example/javatutorial/MainController.java

```diff
@@ -21,4 +29,10 @@ public ResponseEntity<String> readAll(@AuthenticationPrincipal Token token) {
 
         return new ResponseEntity<>("Hello World!", HttpStatus.OK);
     }
+
+    @GetMapping("/hello-world-two")
```

Comment:
I don't like the name of this endpoint, can you please read carefully through
https://restfulapi.net/

Reply:
Also, the second service endpoint is also not correct according to requirements, please check it.